web app/src/main/java/de/tinf15b4/kino/web/views/AccountView.java

```diff
@@ -0,0 +1,240 @@
+            PasswordField newPwCheckField = new PasswordField();
+            newPwCheckField.setCaption("Neues Passwort Wiederholen");
+            newPwCheckRow.addComponent(newPwCheckField);
+            userForm.addComponent(newPwCheckRow);
+
+
+            Button saveButton = new Button("Speichern", e -> trySave(
+                    userField,
+                    emailField,
+                    oldPwField,
+                    newPwField,
+                    newPwCheckField,
+                    currentUser));
+
+            userForm.addComponent(saveButton);
+            saveButton.addStyleName(ValoTheme.BUTTON_PRIMARY);
+
+            saveButton.addShortcutListener(new ShortcutListener(null, ShortcutAction.KeyCode.ENTER, null) {
+                @Override
+                public void handleAction(Object sender, Object target) {
+                    saveButton.click();
+                }
+            });
+
+
+            this.addComponent(userForm);
+        }
+    }
+
+    private void trySave(TextField userField, TextField emailField, PasswordField oldPwField, PasswordField newPwField,
+                         PasswordField newPwCheckField, User currentUser){
+        if (userBean.isUserLoggedIn()){
+
+            String userName = userField.getValue();
+            String email = emailField.getValue();
+            String oldPw = oldPwField.getValue();
+            String newPw = newPwField.getValue();
+            String newCheckPw = newPwCheckField.getValue();
+
+            boolean changeUsername = false;
+            boolean changeEmail = false;
+            boolean changePw = false;
+            boolean valid = true;
+
+            userField.setComponentError(null);
+            emailField.setComponentError(null);
+            oldPwField.setComponentError(null);
+            newPwField.setComponentError(null);
+            newPwCheckField.setComponentError(null);
+
+            // change userName
+            if (!userName.equals(currentUser.getName())){
+                if (userName.length() > 99) {
+                    userField.setComponentError(new UserError("Der angegebene Benutzername ist zu lang"));
+                    valid = false;
+                }
+                else {
+                    if (userName.length() > 2){
+                        changeUsername = true;
+                    }
+                    else {
+                        userField.setComponentError(new UserError("Bitte überprüfen Sie die Eingabe"));
+                        valid = false;
+                    }
+                }
+            }
+
+            // change email
+            if (!email.equals(currentUser.getEmail())){
+                if (email.length() > 99) {
+                    emailField.setComponentError(new UserError("Die angegebene E-Mailadresse" +
+                            " ist zu lang"));
+                    valid = false;
+                }
+                else{
```

Comment:
Uuuuhhhrg did you mangle your autoformatter somehow? This is riddled with basic formatting mistakes.